packages/material/scss/treelist/_layout.scss

```diff
@@ -1 +1,17 @@
 @import "~@progress/kendo-theme-default/scss/treelist/_layout.scss";
+
+@include exports("treelist/layout/material") {
+    .k-treelist {
+        tr.k-footer-template td {
+            border-width: 1px 0;
+        }
+
+        tr.k-footer td {
```

Comment:
I cannot see a case where we use this selector. Could you please show me an example in the TreeList? 

If we are using it, as the CSS is the same, we can add the selector to the next selector.

Reply:
This selector is used in the Kendo UI TreeList for Angular, however I will add it to the next one :)